Content improvements (#354)
apple/swift-org-website: getting-started/_use-cases.md

## Using Swift

<ul class="use-case-list">
  <li class="use-case featured">
    <h3>✨ New to Swift?</h3>
    <p class="description">
      Swift is a great first language if you are just starting your programming journey. For a brief tour of the language, check out this introductory chapter in The Swift Programming Language book.
    </p>

    <a href="https://docs.swift.org/swift-book/documentation/the-swift-programming-language/guidedtour/" class="cta-secondary">Read A Swift Tour</a>
  </li>
</ul>

---

Here are some examples of the many use cases of Swift, in case you want to jump in and start writing some code right away.

<ul class="use-case-list">
  <li class="use-case">
    <h3>Command-line Tool</h3>
    <p class="description">

>       Learn how to create cross-platform command-line tool such as short-lived interactive programs, or long-lived daemon that runs in the background.

Reviewer: ```suggestion
      Learn how to create cross-platform command-line tools such as short-lived interactive programs, or long-lived daemons that run in the background.
```

    </p>

    <a href="/getting-started/cli-swiftpm" class="cta-secondary">Start tutorial</a>
  </li>

  <li class="use-case">
    <h3>Library</h3>
    <p class="description">

>       Learn how to create cross-platform library for sharing reusable code, or modularize large code-base.

Reviewer: ```suggestion
      Learn how to create a cross-platform library for sharing reusable code, or modularize large code-base.
```

    </p>

    <a href="/getting-started/library-swiftpm" class="cta-secondary">Start tutorial</a>
  </li>

  <li class="use-case">
    <h3>Web Service</h3>
    <p class="description">

>       Learn how to create a web-service using a web framework.

Reviewer: ```suggestion
      Learn how to create a web service using a web framework.
```
